### ISSUE TITLE
Schrek.fr

### ISSUE THREAD

**Issue by charl0t** [nkoehring/250kb-club#478]
Adc Schrek.fr
https://yellowlab.tools/result/h36ke8cuyb

**Comment by nkoehring** [comment]
Sorry, but your page loads a 1MB mathjax script. Maybe this can be done optionally on the pages where it is needed? Feel free to reapply again, after this is fixed.

**Comment by charl0t** [comment]
Yes, i change Hugo conf, now MathJax are actived juste for on page thanks
